CDP4RequirementsVerification/Verifiers/RequirementVerificationConfiguration.cs

```diff
@@ -24,19 +24,49 @@
 
 namespace CDP4RequirementsVerification.Verifiers
 {
+    using System.Collections.Generic;
+    using System.Linq;
 
     using CDP4Common.EngineeringModelData;
+    using CDP4Common.Helpers;
 
     /// <summary>
     /// Implements the <see cref="IRequirementVerificationConfiguration"/> for usage as general configuration setting during 
     /// Requirement Verification
     /// </summary>
     public class RequirementVerificationConfiguration : IRequirementVerificationConfiguration
     {
+        /// <summary>
+        /// Backing field for the <see cref="Option"/> property
+        /// </summary>
+        private Option option;
+
         /// <summary>
         /// The <see cref="Option"/> to use during RelationalExpression verification
         /// </summary>
-        public Option Option { get; set; }
+        public Option Option
```

Comment:
please add <remark> to note that NestedParameters property is set as side-effect